src/pages/Login.js

```diff
@@ -0,0 +1,138 @@
+import {useState} from 'react';
+import {AiFillTwitterCircle, AiFillGoogleCircle} from 'react-icons/ai';
+import {RiFacebookCircleFill} from 'react-icons/ri';
+import {useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+import useStoreLogin from '../hooks/useStore_login.js';
```

Comment:
If you change the name of the `useStore `according to my suggestion, please keep in mind to change it here too.🙂

Reply:
Ok